app/models/katello/content_view_version.rb

```diff
@@ -73,8 +73,8 @@ def self.with_library_repo(repo)
 
     def self.for_version(version)
       major, minor = version.to_s.split('.')
-      query = where(:major => major)
-      query.where(:minor => minor) if minor
```

Comment:
@neoseele I should mention, if what @daviddavis and I are thinking is correct, there is no need to change what you have here :)